scripts/release/generate-changelog.js

```diff
@@ -73,53 +75,56 @@ function getChangesList() {
   const lastTagName = command`git describe --tags ${lastTagHash}`.run()
 
   const commits = command`git log ${lastTagName.trimEnd()}..HEAD --pretty=format:%s`.run()
-
   const changesWithEmojis = emojiNameToUnicode(commits)
 
-  const allowedChanges = changesWithEmojis
-    .split('\n')
-    .filter(isNotVersionEntry)
-    .filter(isNotMaintenanceEntry)
-    .sort(byEmojiPriority)
+  let changes = changesWithEmojis.split('\n').filter(isNotVersionEntry)
+  let internalChanges = []
+  let publicChanges = []
+
+  changes.forEach((entry) => {
+    let trimmedEntry = entry.trim()
+    if (INTERNAL_EMOJI_PRIORITY.some((emoji) => trimmedEntry.startsWith(emoji))) {
+      internalChanges.push(entry)
+    } else {
+      publicChanges.push(entry)
+    }
+  })
+
+  internalChanges = internalChanges
+    .sort((a, b) => sortByEmojiPriority(a, b, INTERNAL_EMOJI_PRIORITY))
+    .map((entry) => `- ${entry}`)
+  publicChanges = publicChanges
+    .sort((a, b) => sortByEmojiPriority(a, b, PUBLIC_EMOJI_PRIORITY))
```

Comment:
We may have an encoding issue because ⚗ and ⚡️ are not sorted properly.

Reply:
**❓ question:** It could be nice to have the "Revert" close to the line of the corresponding PR no?‏

Reply:
Yes true some emojies seems to not be sorted properly